boot/env.mli

```diff
@@ -102,6 +102,19 @@ val corelib : t -> Path.t
 (** coq/lib directory, not sure if to keep this *)
 val coqlib : t -> Path.t
 
+val ocamlfind : unit -> string
+
+val docdir : unit -> string
+
+val print_config : ?prefix_var_name:string -> t -> out_channel -> unit
```

Comment:
`print_config` morally belongs to rocqmakefile IMHO, I don't have a concrete proposal to clean it up tho.

Reply:
dune calls it though https://github.com/ocaml/dune/blob/d50209cacaaa8942451b96ff1dc359010113dc06/src/dune_rules/coq/coq_config.mli#L46

Reply:
Yes I'm aware, I'm unsure how to clean things up.